ambry-rest/src/main/java/com.github.ambry.rest/NettyRequest.java

```diff
@@ -52,9 +53,14 @@
  * A wrapper over {@link HttpRequest} and all the {@link HttpContent} associated with the request.
  */
 class NettyRequest implements RestRequest {
+  // If the write of at least {@code bufferWatermark} amount of data is unacknowledged, reading from the channel will be
```

Comment:
Thought that would be redundant since POST is the only one that can have content/data. Do you think it is useful?